packages/nx-plugin/src/generators/init/generator.ts

```diff
@@ -18,15 +18,17 @@ import {
 } from '../../utils/versions';
 import { InitGeneratorSchema } from './schema';
 
+const nxPluginPackageName = '@code-pushup/nx-plugin';
+type PkgJson = Record<string, Record<string, string>>;
```

Comment:
Note: I think there is an official way of typing package.json but I we would need to introduce a new dependency. 😞

Reply:
Found it! 

```ts
import type { PackageJson } from 'nx/src/utils/package-json';
```

Reply:
I had a feeling there was a way when I tried it in the past but TS refused to help me 😅 Good job

Reply:
used the type

Reply:
> used the type

I still see `type PkgJson = Record<string, Record<string, string>>` being used here. Did you push the change?

Reply:
done